chromadb/test/property/strategies.py

```diff
@@ -461,14 +467,21 @@ def recordsets(
 ) -> RecordSet:
     collection = draw(collection_strategy)
 
-    ids = list(
+    ids: IDs = list(
         draw(st.lists(id_strategy, min_size=min_size, max_size=max_size, unique=True))
     )
 
+    n_records = len(ids)
+
+    if len(ids) == 0:
+        n_records += 1
```

Comment:
we will only ever create one record if we draw 0 ids? I think it's better to draw `n` first, then use that length for everything else.

Reply:
Yeah agreed, also in this case you end up only generating ids if there are 0 ids right, i think we should draw n and then flip on ids